docs/DeveloperGuide.md

```diff
@@ -1,29 +1,204 @@
+- deleteMeeting: Delete a meeting in the meeting list by its index.
+- printMeetings: Print all the meetings in the meeting list.
+
+When the user input a meeting command, the Parser will determine which command it is and return a command object.
+One of the meeting commands will be called from `AddMeetingCommand`, `DeleteMeetingCommand` and `ViewMeetingCommand`.
+Inside the command, the execute function will call the corresponding methods inside the MeetingManager and implement on the meeting list.
+
+### Deadline Feature (HuiQi)
+The Deadline Feature allows user to add Deadline objects in a deadline list.
+Deadline objects consists of a description String and a dueDate String.
+The User will also be able to delete deadlines, view deadline list.
+
+When the user input a deadline command, it will be parsed into Parser to determine which deadline command will be called.
+Then, the deadline command will be executed, and the respective method in the DeadlineManager class will be called, which are:
+#### addDeadline
+
+   Adds the deadline object to the ArrayList `deadlines`.
+
+   A deadline object can only be added to the deadlines list if it has a valid description and dueDate String.
+
+   If the addition was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been added.
+
+
+####deleteDeadline
+
+   Removes a deadline object from the ArrayList `deadlines` using the index given.
+   A deadline object can only be removed if the index is a valid index in the list.
+   If the deletion was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been deleted.
+
+3. printDeadlines
+   ![](./uml/ViewDeadlineDiagram.png)
+    Prints the ArrayList `deadlines` through TextUi class.
+
+### Staff Feature 
+The `Staff` Feature allows user to create, read, update, delete (CRUD) `Staff` objects in the list of staffs. 
+`Staff` list is managed by `StaffManager` Class similar to other entity objects. Here are hot it works for each functionality: 
+
+1. add_staff  
+
+    It first checks whether the parameter needed to add `Staff` is already given in the correct format. If it passes the format checking, 
+    the `Parser` object will return `AddStaffCommand` to be executed by `DinerDirector`. Internally, `AddStaffCommand` redirect the requests to 
+    `StaffManager` to add the `Staff` object to the list of `Staff` 
+2. delete_staff 
+
+    It first finds the corresponding `Staff` object to be deleted using its name in the list in `StaffManager`. 
+    If it is found, it will delete the correct corresponding `Staff`. 
+
+3. view_staff
+
+    It prints all of `Staff` objects in `StaffManager`'s staffs. 
+
+### Dish Feature (Zheng Rong)
+
+The Dish feature consists of three functions:
+
+##### Adding dish to list:
+
+![](uml/images/AddDishCommandSequenceDiagram-Add_Dish_Sequence_Diagram.png)
```

Comment:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/88129105/228766230-dfd8b858-32e0-4e56-9e25-a89c84d5b3ab.png">
Do you mean to return before calling execute() ?